.travis.yml

```diff
@@ -1,39 +1,27 @@
-# Config file for automatic testing at travis-ci.org
-
 language: python
 
-python: 3.5
-
 sudo: false
 
 env:
-  matrix:
-    - TOXENV=flake8
-    - TOXENV=isort
-    - TOXENV=py35-dj19-cms33
```

Comment:
@czpython I don't have it in the base .travis.yml file. Currently testing:

python 3.4 cms 3.3 and 3.2, django 1.8 and 1.9
python 2.7 cms 3.3 and 3.2, django 1.8 and 1.9

I'm open to adapt the matrix though we should keep it < 10 tests. What do you suggest? (need to adapt then everywhere)